src/content/docs/new-relic-one/use-new-relic-one/get-started/introduction-new-relic-one.mdx

```diff
@@ -15,35 +15,26 @@ redirects:
   - /node/19491
 ---
 
-[New Relic One](https://one.newrelic.com) is the platform that gives you access to our [Telemetry Data Platform](/docs/telemetry-data-platform/get-started/capabilities/get-know-telemetry-data-platform), our [Full-Stack Observability](/docs/full-stack-observability/monitor-everything/get-started-new-relic-monitoring-tools/get-started-full-stack-observability#h2-start-anywhere) features, and our [Applied Intelligence](/docs/alerts-applied-intelligence) functionality.
+[New Relic One](https://one.newrelic.com) is the platform that gives you access to our [Telemetry Data Platform](/docs/telemetry-data-platform/get-started/capabilities/get-know-telemetry-data-platform), our [Full-Stack Observability](/docs/full-stack-observability/monitor-everything/get-started-new-relic-monitoring-tools/get-started-full-stack-observability#h2-start-anywhere) features, and our [Applied Intelligence](/docs/alerts-applied-intelligence) tools. With New Relic One you can view and interact with all the data from across your entire system more easily. 
```

Comment:
I want to rewrite the last sentence in this first paragraph slightly:

"With New Relic One, you can see and act on all your data throughout your entire system."

Trying to simplify this language a bit. Also, dropped "easily" because I'm allergic to that word :D

Reply:
Thanks, updated!